openpype/lib/vendor_bin_utils.py

```diff
@@ -390,6 +390,76 @@ def get_ffmpeg_tool_path(tool="ffmpeg"):
     return tool_executable_path
 
 
+def _chrome_executable_validation(filepath):
+    """Validate chrome tool executable if can be executed.
+
+    Validation has 2 steps. First is using 'find_executable' to fill possible
+    missing extension or fill directory then launch executable and validate
+    that it can be executed. For that is used '-version' argument which is fast
+    and does not need any other inputs.
+
+    Any possible crash of missing libraries or invalid build should be caught.
+
+    Main reason is to validate if executable can be executed on OS just running
+    which can be issue ob linux machines.
+
+    Note:
+        It does not validate if the executable is really a chrome tool.
+
+    Args:
+        filepath (str): Path to executable.
+
+    Returns:
+        bool: Filepath is valid executable.
+    """
+
+    filepath = find_executable(filepath)
+    if not filepath:
+        return False
+
+    return _check_args_returncode([filepath, "-version"])
```

Comment:
> Install Chrome in your machine (needs to be version 110 of Chrome as version 111 has a bug when trying to set size resolution when using html2image.screenshot) and set $OPENPYPE_CHROME_PATHSto point to the installation directory wherechrome` exists.

Could this technically also log or report the error with that particular version?

Reply:
yeah what would definitely be useful, although I'm not sure if I want to add the complexity of parsing the version string (different in chromium and chrome and who knows other builds) and perhaps introducing some bugs for the small wins of this...
Something like this could maybe work:
```python
version_str = _check_args_returncode([filepath, "-version"])
if not version_str:
    return False

# Check that the major version is not 112 as it has a bug on resolution
integers = re.findall(r"\d+", version_str )
if integers and integers[0] == '112':
    print(f"The major version 112 of Chrome has a bug, update to <112 or 113+.")
    return False

return True
```
what do you think?

Reply:
Ah yes, if version strings returned might differ it might not be worth the trouble. Your check seems fine to me however, hard to tell without test runs.